### ISSUE TITLE
When pipeline include parallel stage，you can not show logs of activity

### ISSUE THREAD

**Issue by wenxin-01** [kubesphere/ks-devops#263]
**Versions used**
KubeSphere: `v3.2.0-alpha.0`

```
pipeline {
  agent any
  stages {
    stage('default-0') {
      parallel {
        stage('test-1') {
          agent none
          steps {
            sh 'echo \'hello tester\''
          }
        }
        stage('test-2') {
          agent none
          steps {
            sh 'echo \'test-2\''
          }
        }
      }
    }
  }
}
```

https://user-images.githubusercontent.com/68640256/134796068-49603852-fc18-4232-9c79-d7985df5dda1.mov

To Reproduce
1、create pipeline
2、run pipeline
3、show logs of activity

/kind bug
/assign @kubesphere/sig-devops 
/priority high

**Comment by LinuxSuRen** [comment]
Excellent catch!

**Comment by wenxin-01** [comment]
issue fixed `nightly-20211008`
